kythe/typescript/testdata/object.ts

```diff
@@ -21,13 +21,27 @@ const Object = {
 
   //- @method defines/binding Method
   //- Method.node/kind function
-  method() {}
+  method() {},
+
+  //- @"'string#literal'" defines/binding SLiteralProperty
+  //- SLiteralProperty.node/kind variable
+  'string#literal': 0,
+
+  //- @"123" defines/binding NLiteralProperty
+  //- NLiteralProperty.node/kind variable
+  123: 'nliteral',
 };
 
 //- @property ref Property
 //- @shortProperty ref ShortProperty
 //- @computed ref ComputedProperty
 const x = Object.property || Object.shortProperty || Object.computed;
 
+//- @"'string#literal'" ref SLiteralProperty
```

Comment:
What I'm worried about is that this will generate a VName with signature
  `whatever.string#literal`
and meanwhile some other code will generate signatures that have other characters at the end with a different meaning, like `#type`.

For another example of this worry, imagine things like
  obj['string.literal'].foo
vs
  obj.string.literal.foo

Reply:
Okay, I see what you're saying. What if we wrap literals in quotes?

Reply:
Yeah that should be fine, I think.

Reply:
Fixed.